docs/changelog.md

```diff
@@ -16,7 +16,10 @@ permalink: /changelog/
   * 🚒 Bug fixed [issue #8464](https://github.com/mozilla-mobile/android-components/issues/8464) - Crash when confirming a prompt that was already confirmed
 
 * **feature-app-links**
-    * Added handling of PackageItemInfo.packageName NullPointerException on some Xiaomi and TCL devices
+  * Added handling of PackageItemInfo.packageName NullPointerException on some Xiaomi and TCL devices
+
+* **service-glean**
+  * 🆙  Updated Glean to version 33.1.2 ([changelog](https://github.com/mozilla/glean/releases/tag/v33.1.2))
```

Comment:
I wonder if it's useful to add a general changelog entry saying something like, "Note that various AndroidX dependencies have been updated in this release."